src/Native/System.Net.Security.Native/pal_gssapi.cpp

```diff
@@ -102,9 +102,11 @@ extern "C" uint32_t NetSecurityNative_DisplayStatus(uint32_t* minorStatus,
     assert(outBuffer != nullptr);
 
     int statusType = isGssMechCode ? GSS_C_MECH_CODE : GSS_C_GSS_CODE;
+    uint32_t messageContent;
     GssBuffer gssBuffer {.length = 0, .value = nullptr};
     uint32_t majorStatus =
-        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, nullptr, &gssBuffer);
+        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, &messageContent, &gssBuffer);
+    (void)messageContent; // unused
```

Comment:
whoa!
This is from OSX header

    GSSAPI_LIB_FUNCTION OM_uint32 GSSAPI_LIB_CALL
    gss_display_status (
	    OM_uint32 *__nonnull minor_status,
	    OM_uint32 status_value,
	    int status_type,
	   __nullable const gss_OID mech_type,
	   OM_uint32 *__nonnull message_content,
      ....


But the rfc uses the same name as the ubuntu. I will change it when I squash it. I'll also try without the ```(void)messageContent```